### PR TITLE
[Feature] add setr cityscapes benchmark

### PR DESCRIPTION
**Pull request by RockeyCoss**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

[Feature] add setr cityscapes benchmark

## Modification

[Feature] add setr cityscapes benchmark

### CODE REVIEW

File: configs/_base_/datasets/cityscapes_768x768.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size=crop_size, pad_val=0, seg_pad_val=255),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_semantic_seg']),
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(2049, 1025),
```
Comment (Junjun2016) [inline configs/_base_/datasets/cityscapes_768x768.py:21]:
```suggestion
        img_scale=(2048, 1024),
```

Should evaluate the results again.

File: configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+_base_ = [
+    '../_base_/models/setr_naive.py',
+    '../_base_/datasets/cityscapes_768x768.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_80k.py'
+]
+norm_cfg = dict(type='SyncBN', requires_grad=True)
```
Comment (Junjun2016) [inline configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py:6]: redundant key.

File: configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            align_corners=False,
+            loss_decode=dict(
+                type='CrossEntropyLoss', use_sigmoid=False, loss_weight=0.4))
+    ],
+    test_cfg=dict(mode='slide', crop_size=crop_size, stride=(512, 512)))
+
+optimizer = dict(
+    lr=0.01,
+    weight_decay=0.0,
+    paramwise_cfg=dict(custom_keys={'head': dict(lr_mult=10.)}))
```
Comment (Junjun2016) [inline configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py:66]: Should remove redundant keys from `../_base_/models/setr_naive.py`.

File: configs/setr/setr_pup_vit-large_8x1_768x768_80k_cityscapes.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            align_corners=False,
+            loss_decode=dict(
+                type='CrossEntropyLoss', use_sigmoid=False, loss_weight=0.4))
+    ],
+    test_cfg=dict(mode='slide', crop_size=crop_size, stride=(512, 512)))
+
+optimizer = dict(
+    lr=0.01,
+    weight_decay=0.0,
+    paramwise_cfg=dict(custom_keys={'head': dict(lr_mult=10.)}))
```
Comment (Junjun2016) [inline configs/setr/setr_pup_vit-large_8x1_768x768_80k_cityscapes.py:66]: Should also remove redundant keys from the schedule.

File: configs/setr/setr_mla_vit-large_8x1_768x768_80k_cityscapes.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
+_base_ = [
```
Comment (Junjun2016) [inline configs/setr/setr_mla_vit-large_8x1_768x768_80k_cityscapes.py:1]: rename `configs/setr/setr_mla_vit-large_8x1_768x768_80k_cityscapes.py` to `configs/setr/setr_vit-large_mla_8x1_768x768_80k_cityscapes.py`

File: configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
+_base_ = [
```
Comment (Junjun2016) [inline configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py:1]: rename `configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py` to `configs/setr/setr_vit-large_naive_8x1_768x768_80k_cityscapes.py`

File: configs/setr/setr_pup_vit-large_8x1_768x768_80k_cityscapes.py [hunk 7]
```diff
@@ -0,0 +1,68 @@
+_base_ = [
```
Comment (Junjun2016) [inline configs/setr/setr_pup_vit-large_8x1_768x768_80k_cityscapes.py:1]: rename `configs/setr/setr_pup_vit-large_8x1_768x768_80k_cityscapes.py` to `configs/setr/setr_vit-large_pup_8x1_768x768_80k_cityscapes.py`

**Review by Junjun2016 — APPROVED** [review]

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**